src/types/BatchClaims.sol

```diff
@@ -20,3 +20,13 @@ struct BatchClaim {
     string witnessTypestring; // Witness typestring appended to existing typestring.
     BatchClaimComponent[] claims; // The claim token IDs, recipients and amounts.
 }
+
+using BatchClaimsLib for BatchClaim global;
```

Comment:
Is the global using necessary or can we scope it to the areas where it's specifically needed?